src/main/java/bio/terra/service/search/SearchService.java

```diff
@@ -57,16 +62,30 @@ public SearchService(BigQueryPdao bigQueryPdao, RestHighLevelClient client) {
         this.client = client;
     }
 
+    private String uuidToIndexName(UUID id) {
+        return String.format("idx-%s", id);
+    }
+
+    private String translateToTim(String sql) {
```

Comment:
It's a minor point, but I don't think this translates to TIM. It encodes any TIM names in the query.

I've been thinking that maybe the encoding operation is not necessarily TIM specific (although TIM is the only current use case) so maybe the notion here is that we need to encode any name, not just TIM ones.